src/nectarchain/dqm/start_calib_thermal_test.py

```diff
@@ -0,0 +1,213 @@
+import os
+import sys
+
+from matplotlib import pyplot as plt
+
+#from multiprocessing import Process
+
+import time
+
+from ctapipe.io import EventSource, EventSeeker
+
+
+from mean_waveforms import MeanWaveForms_HighLowGain
+from mean_camera_display import MeanCameraDisplay_HighLowGain
+from charge_integration import ChargeIntegration_HighLowGain
+from trigger_statistics import TriggerStatistics
+from camera_monitoring import CameraMonitoring
+
+import argparse
+
+# Create an ArgumentParser object
+parser = argparse.ArgumentParser(description='My script')
+parser.add_argument('-p', '--plot',
+                     action='store_true',
+                     help='Enables plots to be generated')
+parser.add_argument('-n', '--noped',
+                     action='store_true',
+                     help='Enables plots to be generated')
+parser.add_argument('input_paths', nargs='+', help='Input paths')
+args = parser.parse_args()
+
+#Reading arguments, paths and plot-boolean
+path = args.input_paths[0]
+print("Input file paths:")
+print(path)
+for arg in args.input_paths[1:]:
+    print(arg)
+PlotFig = args.plot
+noped = args.noped
+print("Plot:", PlotFig)
+
+#NectarPath
+NectarPath = str(os.environ['NECTARDIR'])
+
+
+
+def GetName(RunFile):
+    name = RunFile.split('/')[-1]
+    name = name.split('.')[0] + '_' + name.split('.')[1]# + '_' +name.split('.')[2]
+    print(name)
+    return name
+
+def CreateFigFolder(name, type):
+    if(type == 0):
+        folder = 'Plots'
+
+    ParentFolderName = name.split('_')[0] + '_' + name.split('_')[1]
+    ChildrenFolderName = './' + ParentFolderName +'/' + name + '_calib'
+    FolderPath = NectarPath + 'output/%s/%s/' %(ChildrenFolderName, folder)
+
+    if not os.path.exists(FolderPath):
+        os.makedirs(FolderPath)
+
+    return ParentFolderName, ChildrenFolderName, FolderPath
+
+
+start = time.time()
+
+#INITIATE
+#######################################################################################################################
+path = path
+print(path)
+cmap = 'gnuplot2'
+
+#Read and seek
+reader=EventSource(input_url=path)
+seeker = EventSeeker(reader)
+reader1 = EventSource(input_url=path, max_events=1)
+#print(reader.file_list)
+
+name = GetName(path)
+ParentFolderName, ChildrenFolderName, FigPath = CreateFigFolder(name, 0)
+ResPath = NectarPath + 'output/%s/%s' %(ChildrenFolderName, name)
+#######################################################################################################################
+
+
+
+
+                                                  ########################
+
+
+
+
+
+
+
+#LIST OF PROCESSES TO RUN
+#######################################################################################################################
+a = TriggerStatistics(0)
+b = MeanWaveForms_HighLowGain(0) #0 is for high gain and 1 is for low gain
```

Comment:
It would be better to rely here on https://github.com/cta-observatory/ctapipe_io_nectarcam/blob/master/src/ctapipe_io_nectarcam/constants.py, such as:

```
b = MeanWaveForms_HighLowGain(ctapipe_io_nectarcam.constants.LOW_GAIN)
c = MeanWaveForms_HighLowGain(ctapipe_io_nectarcam.constants.HIGH_GAIN)
```
to avoid hardcoding things as much as possible.

Reply:
Also, this script should be named differently, such as `startCalibThermalTest.py` for instance. As such, `pytest` triggered by the CI is failing, see https://github.com/cta-observatory/nectarchain/actions/runs/4308630949/jobs/7515112484.

Reply:
This script was removed, but I implemented the first coment in the start_calib.py script